core/vm/jump_table.go

```diff
@@ -77,6 +78,13 @@ func newSubnetEVMInstructionSet() JumpTable {
 	return validate(instructionSet)
 }
 
+func newDUpgradeInstructionSet() JumpTable {
+	instructionSet := newSubnetEVMInstructionSet()
+	enable3855(&instructionSet) // PUSH0 instruction
+	enable3860(&instructionSet) // Limit and meter initcode
+	return validate(instructionSet)
+}
```

Comment:
Can we move the DUpgrade and SubnetEVM instruction set functions below `validate(...)` ?

Reply:
done